SDWebImage/Core/NSData+ImageContentType.h

```diff
@@ -38,7 +38,7 @@ static const SDImageFormat SDImageFormatSVG       = 8;
  *
  *  @return the image format as `SDImageFormat` (enum)
  */
-+ (SDImageFormat)sd_imageFormatForImageData:(nullable NSData *)data;
++ (SDImageFormat)sd_imageFormatForImageData:(nullable NSData *)data NS_SWIFT_NAME(sd_imageFormat(from:));
```

Comment:
For consistency with the Objective-C version maybe it's better to replace "from" with "for" here?
If you still think that "from" is better then maybe let's also update the Objective-C to use the same words?